doc/whats_new.rst

```diff
@@ -71,6 +71,8 @@ Changelog
 
 - Add :meth:`mne.io.Raw.reorder_channels`, :meth:`mne.Evoked.reorder_channels`, etc. to reorder channels, by `Eric Larson`_
 
+- Add ability to exclude components interactively by clicking on their labels in :meth:`mne.preprocessing.ICA.plot_properties` by `Mikołaj Magnuski`_
```

Comment:
You mean :meth:`mne.preprocessing.ICA.plot_components` I think.

Reply:
right :)